MediaBrowser.Providers/Manager/ProviderManager.cs

```diff
@@ -235,14 +236,8 @@ public async Task<IEnumerable<RemoteImageInfo>> GetAvailableRemoteImages(BaseIte
 
             var preferredLanguage = item.GetPreferredMetadataLanguage();
 
-            var languages = new List<string>();
-            if (!query.IncludeAllLanguages && !string.IsNullOrWhiteSpace(preferredLanguage))
-            {
-                languages.Add(preferredLanguage);
-            }
-
             // TODO include [query.IncludeAllLanguages] as an argument to the providers
```

Comment:
```suggestion
```

I think this `TODO` has been implemented now

Reply:
No it hasn't